### PR TITLE
Update Object Lists

### PR DESCRIPTION
**Pull request by SalleeMatthew**
Updates object lists when an object instance is updated or purchased.

### CODE REVIEW

File: spa/src/pages/world-browser/WorldBrowserPage.vue [hunk 1]
```diff
@@ -576,15 +581,16 @@ export default Vue.extend({
         this.eventNodeMap.get(eventNode.name).push(eventNode);
       }
     },
-    startSocketListeners(): void {
+     startSocketListeners(): void {
       this.$socket.on("VERSION", event => this.onVersion(event));
+      this.$socket.on("update-object", () => setTimeout(this.onSharedObjectEvent, 50));
+      this.$socket.on("SO", event => this.onSharedObjectEvent(event));
```
Comment (smile0711) [inline spa/src/pages/world-browser/WorldBrowserPage.vue:587]: Why was this moved to here?

Comment (SalleeMatthew) [reply spa/src/pages/world-browser/WorldBrowserPage.vue:587]: Dropping and picking up objects uses the SO event. So, this only affected users in 2d if the user had went 3d prior to either one happening. The SO event listener needs to be started when the user joins the room rather than only when the user goes 3d.

Comment (smile0711) [reply spa/src/pages/world-browser/WorldBrowserPage.vue:587]: Ok, thanks
